node/sc/subbridge_test.go

```diff
@@ -105,8 +106,8 @@ func TestSubBridge_basic(t *testing.T) {
 func TestSubBridge_removePeer(t *testing.T) {
 	// Create a test SubBridge (it may have 0 peers)
 	sBridge := testNewSubBridge(t)
-	defer sBridge.chainDB.Close()
 	defer sBridge.removeData(t)
+	defer sBridge.chainDB.Close()
```

Comment:
Is there any reason for changing the order? I think DB should be closed before removing the data directory.

Reply:
Deferred function calls are pushed onto a stack. When a function returns, its deferred calls are executed in last-in-first-out order.
Close is called before removing after this change.

Reply:
@winnie-byun Oh, you are right. Thanks!